Product/Production/Adapters/DocumentQuery_a0/src/main/java/gov/hhs/fha/nhinc/docquery/deferred/adapter/AdapterDeferredResponseOptionQuerySecured.java

```diff
@@ -37,10 +37,10 @@
 
 public class AdapterDeferredResponseOptionQuerySecured implements AdapterDeferredResponseOptionQueryRequestSecuredPortType{
 
-
     @Override
-    public RespondingGatewayCrossGatewayQuerySecureResponseType respondingGatewayCrossGatewayQueryDeferredSecured(
-        RespondingGatewayCrossGatewayQuerySecureRequestType message) {
+    public RespondingGatewayCrossGatewayQuerySecureResponseType respondingGatewayCrossGatewayQueryDeferredSecuredRequest(
+        RespondingGatewayCrossGatewayQuerySecureRequestType arg0) {
```

Comment:
Optional: make sure we use appropriate name for variable instead of arg0.  Also, I assume this feature will implement later